client/src/views/organizations/creation/CreateOrganizationSaas.vue

```diff
@@ -6,6 +6,13 @@
       v-show="step === Steps.BmsLogin"
       @login-success="onLoginSuccess"
       @close-requested="$emit('closeRequested', false)"
+      @forgotten-password-clicked="step = Steps.BmsForgotPassword"
+    />
+    <bms-forgot-password
+      v-show="step === Steps.BmsForgotPassword"
```

Comment:
Replace by `v-if` here, otherwise `BmsForgotPasswordGetEmail` gets mounted very early, which means it won't get the focus. With the `v-if`, it will get mounted at the right time and get the focus properly.

Reply:
done